aggregator/handler/config.go

```diff
@@ -141,25 +144,31 @@ func (c *connectionConfiguration) NewConnectionOptions(scope tally.Scope) Connec
 	return opts
 }
 
-type partitionedHandlerConfiguration struct {
-	// Name of the partitioned backend.
+type shardedHandlerConfiguration struct {
+	// Name of the sharded backend.
 	Name string `yaml:"name"`
 
-	// Backend server partitions.
-	Partitions []BackendServerPartition
+	// Backend server shards.
+	Shards []BackendServerShard `yaml:"shards"`
 
-	// Total queue size across all partitions.
+	// Total queue size across all shards.
 	QueueSize int `yaml:"queueSize"`
 
 	// Connection configuration.
 	Connection connectionConfiguration `yaml:"connection"`
+
+	// Disable validation (dangerous but useful for testing and staging environments).
+	DisableValidation bool `yaml:"disableValidation"`
 }
 
-func (c *partitionedHandlerConfiguration) NewHandler(
+func (c *shardedHandlerConfiguration) NewHandler(
+	totalShards int,
 	instrumentOpts instrument.Options,
 ) (aggregator.Handler, error) {
-	if err := c.validatePartitions(); err != nil {
-		return nil, err
+	if !c.DisableValidation {
+		if err := c.validateShards(totalShards); err != nil {
```

Comment:
nit: handle `if !c.DisableValidation` in validateShards()?

Reply:
Done